File: milestone 2/python_modules/module00/ex5/ft_count_harvest_iterative.py
def ft_count_harvest_iterative():
    counter = 0

    print("Days until harvest: ", end="")
    days_till_harvest = int(input())

    while counter < days_till_harvest:
        counter += 1
        print(f"Day {counter}")
    print("Harvest time!")


ft_count_harvest_iterative()

File: milestone 2/python_modules/module00/ex5/ft_count_harvest_iterative.cpp
#include <iostream>

void	ft_count_harvest_iterative()
{
	int days_till_harvest, counter = 0;

	std::cout << "Days until harvest: ";
	std::cin >> days_till_harvest;
	while (counter < days_till_harvest)
	{
		counter++;
		std::cout << "Day " << counter << "\n";
	}
	std::cout << "Harvest time!\n";
}

int		main(void)
{
	ft_count_harvest_iterative();
	return 0;
}